indico/htdocs/sass/modules/event_display/_conferences.scss

```diff
@@ -161,11 +157,17 @@
                 font-size: 1.5em;
                 font-weight: normal;
 
-                &[class^="icon-"]:before,
-                &[class*=" icon-"]:before {
+                &[class^='icon-']::before,
+                &[class*=' icon-']::before {
                     margin-right: 5px;
                 }
             }
+
+            h4 {
```

Comment:
We discussed that this would be a nice addition to our conference page layout template. Currently we don't account on an `h4` but I believe we should standardize how the `h4`s are displayed the same way we do with the `h3`s. That's why I'd like to suggest to move this code to `_conferences.scss` inside the `.header-row` selector.